roles/func_accounts/meta/main.yml

```diff
@@ -1,2 +1,2 @@
 dependencies:
-  - { role: anaconda, tags: anaconda }
+  - { role: ngi_pipeline, tags: ngi_pipeline }
```

Comment:
I wonder, does it ever happen that one runs a specific role from miarka-provision? It seems to me that the entire install.yml playbook is run as a whole each time.